README.md

```diff
@@ -462,14 +442,21 @@ From within the support folder, edit your .env to include your newly acquired cr
 Redeploy the support microservice, the support feature should now be accessible through the portal.
 
 ```
-$ bx dev deploy
+$ ibmcloud dev deploy
 ```
 
+# Troubleshooting
+
+1. Trouble with IBM Clouc Cli? [Check Here](https://console.bluemix.net/docs/cli/ts_createapps.html#troubleshoot)
```

Comment:
typo with `Clouc`

Reply:
Cli -> CLI

Reply:
Fixed the typos listed. 
I did another scan of the document. Did not spot another one.